### ISSUE TITLE
How to do "direct triangulation" on baseline "Single"

### ISSUE THREAD

**Issue by Zhaoxuyu** [microsoft/multiview-human-pose-estimation-pytorch#34]
I have a question on chaptar 6.3, table 2.

I want to know how you lifting 3D Pose from 2D Pose for baseline "Single". The paper says "direct triangulation".
 
According to my understanding, the input of "direct triangulation" is multiple view.

But for output of baseline "Single", it is only single view, how to do "direct triangulation" for it?

**Comment by haibo-qiu** [comment]
@Zhaoxuyu 
Yes, the "direct triangulation" takes multi-view images as input.

The H36M provides four views, but the baseline "Single"  uses separated view images for training without any fusion. Then we utilize the trained "Single" model to obtain the heatmaps of four views images, which will be taken by "Triangulation" to get the 3D pose.

**Comment by 99sun99** [comment]
So you didn't use the lib/multiview/triangulate.py   in your training process?  why? I think you write it for a reason~
